.github/workflows/main.yaml

```diff
@@ -0,0 +1,105 @@
+name: AndcultureCode.Cli
```

Comment:
Dig it! I'll have to look at the documentation some time to see what all is available. Looks pretty nice with matrix builds

Reply:
For sure - The way the actions are setup makes it super easy to plug & play different configurations in a single job, which was more of a pain in Travis (could only use one main image as a base, and install any other dependencies manually..). Travis had matrix builds, too, but you'd run out your free credits real quickly with any sort of advanced matrix setup.